content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst

```diff
@@ -0,0 +1,184 @@
+
+Next, configure an :guilabel:`Order Deadline`, which is the date by which the vendor must confirm
+their agreement to supply the products.
+
+.. note::
+   After the :guilabel:`Order Deadline` is exceeded, the |RFQ| is marked as late, but the products
+   can still be ordered.
+
+:guilabel:`Expected Arrival` is automatically calculated based on the :guilabel:`Order Deadline`
+and vendor lead time. Tick the checkbox for :guilabel:`Ask confirmation` to ask for signage at
+delivery.
+
+With the :doc:`Storage Locations feature
+<../../inventory/warehouses_storage/inventory_management/use_locations>` activated,
+the :guilabel:`Deliver to` field appears, with options for the order shipment.
+
+Select the receiving warehouse address here, or select :guilabel:`Dropship` to indicate that this
+order is to be shipped directly to the end customer. When :guilabel:`Dropship` is selected, the
+:guilabel:`Dropship address` field is enabled. Contact names auto-populate here from the
+**Contacts** app.
+
+Products tab
+~~~~~~~~~~~~
+
+In the :guilabel:`Products` tab, add the products to be ordered. Click :guilabel:`Add a product`,
+and type in the product name, or select the item from the drop-down menu.
+
+To create a new product and add it, type the new product name in the :guilabel:`Product` column,
+select :guilabel:`Create [product name]` from the resulting drop-down menu, and manually add the
+unit price. Or, select :guilabel:`Create and edit...` to be taken to the product form for that new
+item.
+
+:guilabel:`Catalog` can also be selected to navigate to a product menu from the chosen vendor. From
+here, products can be added to the cart.
+
+.. note::
+  To make adjustments to products and prices, access the product form by clicking the
+  :icon:`oi-arrow-right` :guilabel:`(right arrow)` icon that becomes available upon hovering over
+  the :guilabel:`Product` name.
+
+Send |RFQ|
+----------
+
+Clicking :guilabel:`Send by Email` reveals a :guilabel:`Compose Email` pop-up window, with a
+:guilabel:`Purchase: Request for Quotation` template loaded, ready to send to the vendor's email
+address (configured in the **Contacts** app).
+
+After crafting the desired message, click :guilabel:`Send`. Once sent, the |RFQ| moves to the
+:guilabel:`RFQ Sent` stage.
+
+Clicking :guilabel:`Print RFQ` downloads a PDF of the |RFQ|.
+
+Confirm order
+-------------
+
+Clicking :guilabel:`Confirm Order` directly transforms the |RFQ| into an active |PO|.
+
+.. tip::
+   Odoo tracks communications on each order through the chatter of the |PO| form. This shows the
+   emails sent between the user and the contact, as well as any internal notes and activities.
+   Messages, notes, and activities can also be logged on the chatter.
+
+Once an |RFQ| is confirmed, it creates a |PO|.
+
+On the new |PO|, the :guilabel:`Order Deadline` field changes to :guilabel:`Confirmation Date`, which
+displays the date and time the user confirmed the order.
```

Comment:
Need to break this line a little earlier, as "which" ends up being over 100 chars.
```suggestion
On the new |PO|, the :guilabel:`Order Deadline` field changes to :guilabel:`Confirmation Date`,
which displays the date and time the user confirmed the order.
```